docs/FAQ.md

```diff
@@ -12,6 +12,7 @@ The FAQ predated the introduction of the rest of the documentation. If something
 ## Configuration
 * [What is the difference between Gopkg.toml (the "manifest") and Gopkg.lock (the "lock")?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
 * [How do I constrain a transitive dependency's version?](#how-do-i-constrain-a-transitive-dependency-s-version)
+* [How do I change change the version of a dependecy?](#how-do-i-change-change-the-version-of-a-dependecy)
```

Comment:
oops! Found one more thing, sorry I didn't catch it the first time. We have a stutter "change change" in the title of this question that needs to be fixed here and in the header.

Reply:
Sorry. My bad, I should have double checked that.Pushed those changes, please do take a look.